zebrad/tests/acceptance.rs

```diff
@@ -197,7 +207,7 @@ fn seed_args() -> Result<()> {
 #[test]
 fn start_no_args() -> Result<()> {
     zebra_test::init();
-    let (tempdir, _guard) = tempdir(true)?;
+    let (tempdir, _guard) = tempdir(ConfigMode::Persistent)?;
```

Comment:
Suggestion: Now we have a separate persistent test, we don't need to use persistent for an unrelated test.

```suggestion
    let (tempdir, _guard) = tempdir(ConfigMode::Ephemeral)?;
```